docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -6,19 +6,22 @@
 
 preview::[]
 
+Updates the draft `filtering` configuration of a connector and marks the draft validation state as `edited`. The draft filtering can be then activated after a successful validation of the draft by the {enterprise-search-ref}/connectors.html[Elastic connector service].
 
-Updates the `filtering` configuration of a connector. Learn more about filtering in the {enterprise-search-ref}/sync-rules.html[sync rules] documentation.
+Learn more about filtering for connectors in the {enterprise-search-ref}/sync-rules.html[sync rules] documentation.
 
 [[update-connector-filtering-api-request]]
 ==== {api-request-title}
 
 `PUT _connector/<connector_id>/_filtering`
+`PUT _connector/<connector_id>/_filtering/_activate`
 
 [[update-connector-filtering-api-prereq]]
 ==== {api-prereq-title}
 
 * To sync data using self-managed connectors, you need to deploy the {enterprise-search-ref}/build-connector.html[Elastic connector service] on your own infrastructure. This service runs automatically on Elastic Cloud for native connectors.
 * The `connector_id` parameter should reference an existing connector.
+* Draft filtering needs to be validated by the Elastic connector service before its activation. The `draft.validation.state` needs to be `valid`.
```

Comment:
"Draft filtering needs to be validated by the Elastic connector service" reads _passively_, but as a user what do I actively have to do here to ensure this validation happens?

Reply:
same as with configuration I replied above:

They need to have the configured service running:

- for native connectors the schema will be populated by the connector service automatically
- for self-managed connectors, you need to populate config.yml and start the service

The upon adding new draft filtering, the framework will pick it up, validate (note advanced rules could be validating for couple of mins 💀 ) and then the draft will be in `valid` state. 

then `_filtering/_activate` endpoint will work, otherwise it will throw and error saying:
```
Filtering draft needs to be validated by the connector service before activation. Current filtering draft validation state [edited] is not equal to [valid].
```

Reply:
OK cool I mean I think you're covered with "the connector service must be running". The API calls make no sense without the connector service running anyway, so I think we can

Re. the error message I would also be explicit saying something like "Make sure connector service is running."